testing/fixtures/questions.json

```diff
@@ -2276,7 +2411,7 @@
       "help_lang4": "",
       "help_lang5": "",
       "text_lang1": "Text?",
-      "text_lang2": "￼Text?",
+      "text_lang2": "\ufffcText?",
```

Comment:
Somehow your json does not use `ensure_ascii=False`.

Reply:
Ive used
```bash
python manage.py dumpdata questions | python -m json.tool --indent 2 > questions.json
``
and after that, I manually reverted these unicode escaped strings, but forgot this one.
This symbol should be removed in the original fixture anyway or?